pilz_trajectory_generation/test/integrationtest_command_list_manager.cpp

```diff
@@ -203,6 +203,48 @@ TEST_F(IntegrationTestCommandListManager, concatThreeSegments)
   EXPECT_LT(fabs((t1_2_3-t1-t2-t3)), 0.4);
 }
 
+
+/**
+ * @brief Tests if times are strictly increasing with selective blending
+ *
+ * Test Sequence:
+ *    1. Generate request with three trajectories where only the first has a blend radius.
+ *    1. Generate request with three trajectories where only the second has a blend radius.
+ *    1. Generate request with three trajectories where only the third (last) has a blend radius.
+ *
```

Comment:
Oops, I commented on the resolved conversation:
I still think we don't need the three cases.
* There is already a test case "lastBlendingRadiusNonZero"
* the other two can be combined into: "0.0 - 0.1 - 0.0" since only the transitions matter.